plugins/ccgx/ccgx.quirk

```diff
@@ -10,6 +10,7 @@ GType = FuCcgxHpiDevice
 
 [DeviceInstanceId=USB\VID_04B4&PID_521A&SID_1F00&APP_6D64]
 ImageKind = dual-asymmetric
+DeviceName = ThinkPad USB-C Dock Gen2 PD Controller
```

Comment:
Isn't this the purpose of the `Name` quirks key?

Reply:
Fixed at [https://github.com/fwupd/fwupd/pull/2013/commits/d40f67f4c8af3ebfc2342f155d367eb7b6b73e15](url)